common/src/main/java/org/apache/seata/common/loader/EnhancedServiceLoader.java

```diff
@@ -616,8 +525,17 @@ private List<ExtensionDefinition<S>> findAllExtensionDefinition(ClassLoader load
             return extensionDefinitions;
         }
 
+        private static Class getCompatibleService(Class originType) {
+            String ioSeataType = originType.getName().replace(APACHE_SEATA_PACKAGE_NAME, IO_SEATA_PACKAGE_NAME);
+            try {
+                return Class.forName(ioSeataType);
+            } catch (ClassNotFoundException e) {
+                return null;
+            }
+        }
+
 
-        private void loadFile(String dir, ClassLoader loader, List<ExtensionDefinition<S>> extensions)
+        private void loadFile(String dir, Class type, ClassLoader loader, List<ExtensionDefinition<S>> extensions)
```

Comment:
It seems that the 'type' parameter is not being used.

Reply:
it`s use for load for SPI Compatible